person.rb

```diff
@@ -27,4 +27,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(date, person)
+    Rental.new(date, person, self)
```

Comment:
- [ ] In the `person.rb` you also need `rentals` as an instance variable just like in the `book.rb` to make the association. Also, getters and setters should be present for `rentals`.